src/ros_filter.cpp

```diff
@@ -1342,56 +1346,59 @@ namespace RobotLocalization
 
         // If the worldFrameId_ is the odomFrameId_ frame, then we can just send the transform. If the
         // worldFrameId_ is the mapFrameId_ frame, we'll have some work to do.
-        if (filteredPosition.header.frame_id == odomFrameId_)
```

Comment:
All the following changes are indentation changes due to the addition of the if (publishTransform_)